utils/constants.py

```diff
@@ -33,3 +33,11 @@
 '''
 
 NEW_CLIENT_INITIALIZATION_TOPIC = 'client_ids'
+
+CONFIGURATION_FEDERATED = 'federated'
+CONFIGURATION_CENTRALIZED = 'centralized'
+
+AGGREGATITON_SCHEME_AVERAGE = 'average'
+
+NETWORK_STALE = 'stale'
+NETWORK_NEW = 'new'
```

Comment:
I think these are better as enumerators, look at mqtt_helper for an example of an enum.